Force linking against share object libraries, not static ones.--- a/cmake/OpenMSBuildSystem_externalLibs.cmake
+++ b/cmake/OpenMSBuildSystem_externalLibs.cmake
@@ -48,12 +48,18 @@
 ##             compiler-wise: use the same compiler for contrib and OpenMS!
 
 #set which library extensions are preferred (we want static libraries)
-if(NOT MSVC)
-	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a;.so")
-endif()
-if (APPLE)
-	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a;.dylib")
-endif()
+# if(NOT MSVC)
+# 	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a;.so")
+# endif()
+# if (APPLE)
+# 	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a;.dylib")
+# endif()
+# 
+
+# We want to FORCE *.so to be used, not *.a.
+
+set(CMAKE_FIND_LIBRARY_SUFFIXES ".so")
+
 
 OPENMS_CHECKLIB(CONTRIB_XERCESC "xerces-c_3;xerces-c_static_3;libxerces-c;xerces-c" "xerces-c_3D;xerces-c_static_3D;libxerces-c;xerces-c" "xerces_c")
 
